repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/addupgradebootentry.py

```diff
@@ -35,17 +38,20 @@ def add_boot_entry():
         )
 
 
-def _remove_old_upgrade_boot_entry(kernel_dst_path):
+def _remove_old_upgrade_boot_entry(kernel_dst_path, config=None):
     """
     Remove entry referring to the upgrade kernel.
 
     We have to ensure there are no duplicit boot entries. Main reason is crash
     of zipl when duplicit entries exist.
     """
-    run([
+    cmd = [
         '/usr/sbin/grubby',
         '--remove-kernel', '{0}'.format(kernel_dst_path)
-    ])
+    ]
+    if config:
+        cmd += '-c' + config
```

Comment:
`cmd` is a list not a string, this doesn't seem a good idea also doesn't there need to be a space between -c and config?

Reply:
I meant `cmd += ['-c' + config]` of course :) Thanks for the space reminder too...

Reply:
but it won't do what I wanted anyway :)